### DISCUSSION TITLE
generate synthetic data for multi-node DDP

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1286] [Q&A]
Hi there,
In the tutorial script [unet_training_ddp.py](https://github.com/Project-MONAI/tutorials/blob/main/acceleration/distributed_training/unet_training_ddp.py), it seems all local_rank = 0 at each node will generate synthetic data to the same directory when we do multi-node training. Is it necessary and safe? Or shall we only let global_rank = 0 to generate data?

**Comment by KumoLiu** [comment]
Hi @hw-ju, when doing multi-node training, I think the synthetic data will be generated to the same directory but on different nodes, so it's necessary.
Hope it can help you, thanks!
